<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QWidget" name="WelcomeScreen">
     <widget class="QLabel" name="WelcomeLabel_WelcomeScreen">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to Educational Management System</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton_WeclomeScreen">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SignupButton_WelcomeScreen">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginScreen">
     <widget class="QPushButton" name="LoginButton_LoginScreen">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackButton_LoginScreen">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="EmailLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="EmailTextEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="PasswordTextEdit">
      <property name="geometry">
       <rect>
        <x>133</x>
        <y>100</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SignUpScreen"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
